.flake8

```diff
@@ -0,0 +1,5 @@
+# Can't yet be moved to the pyproject.toml due to https://gitlab.com/pycqa/flake8/-/issues/428#note_251982786
+[flake8]
+max-line-length = 88
```

Comment:
I am also more happy with max line legnth=120. We have that in squidpy as well

Reply:
I opened https://github.com/theislab/scanpy/issues/1694 to discuss this. Insert Bernie meme